src/Avatar/__stories__/Avatar.stories.js

```diff
@@ -1,43 +1,67 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
-import { withKnobs, text, select } from '@storybook/addon-knobs';
+import { number, text, withKnobs } from '@storybook/addon-knobs';
 
 import { Avatar } from '../..';
 import AvatarReadme from '../README.md';
 
 storiesOf('Avatar', module)
   .addDecorator(withKnobs)
+  .addParameters({
+    readme: {
+      includePropTables: [Avatar], // won't work right now because of wrapped styled-comp https://github.com/tuchk4/storybook-readme/issues/177
+    },
+  })
   .addParameters({
     readme: {
       // Show readme before story
       content: AvatarReadme,
     },
   })
```

Comment:
Does it normal you've got two different parameters here ? Maybe you can do that instead: 
```js
.addParameters({
    readme: {
      // Show readme before story
      content: AvatarReadme,
      includePropTables: [Avatar], 
      // won't work right now because of wrapped styled-comp https://github.com/tuchk4/storybook-readme/issues/177
    },
  })
```

Reply:
Thanks for the feedback (fyi it is `Is it normal`)